### PR TITLE
Demonstration of Code Review

### PR DESCRIPTION
**Pull request by Xiaoming94**
asdasd

### CODE REVIEW

File: hello_test.cc [hunk 1]
```diff
@@ -10,13 +10,10 @@
 
 // }
 
-TEST(testing, fail)
-{
-	EXPECT_EQ(false, true);
```
Comment (Xiaoming94) [inline hello_test.cc:15]: What is this bro=?

**Review by risnew — COMMENTED** [review]
nice

File: hello_test.cc [hunk 2]
```diff
@@ -10,13 +10,10 @@
...
-TEST(testing, succeed)
-{
-	EXPECT_EQ(true, true);
+TEST(testing, fail) {
+  std::cout << "Hello test" << std::endl;
+  EXPECT_EQ(false, true);
+  EXPECT_THAT(false, Eq(true));
 }
 
+TEST(testing, succeed) { EXPECT_EQ(true, true); }
```
Comment (risnew) [inline hello_test.cc:19]: comment on 19

File: hello_test.cc
Comment (Veera-Velugubantla) [inline hello_test.cc:1]: Hello

File: hello_test.cc [hunk 3]
```diff
@@ -10,13 +10,10 @@
 
 // }
 
-TEST(testing, fail)
-{
-	EXPECT_EQ(false, true);
-}
-
-TEST(testing, succeed)
```
Comment (vbharu-96) [inline hello_test.cc:18]: testing

File: hello_test.cc [hunk 4]
```diff
@@ -10,13 +10,10 @@
...
-{
-	EXPECT_EQ(true, true);
+TEST(testing, fail) {
+  std::cout << "Hello test" << std::endl;
+  EXPECT_EQ(false, true);
```
Comment (CsabaBeres) [inline hello_test.cc:15]: This is false

File: hello_test.cc
Comment (CsabaBeres) [reply hello_test.cc:1]: Yeah, @Veera-Velugubantla

**Review by CsabaBeres — COMMENTED** [review]
Kamuhab is here

**Review by vbharu-96 — COMMENTED** [review]
see if notification received

**Pull request merged**